data/glossaries/external.yml

```diff
@@ -204,6 +204,9 @@
     A diff is the difference in changes between two commits, or saved changes.
     The diff will visually describe what was added or removed from a file since
     its last commit.
+- term: directory
+  description: >-
+    Computer files are cataloged in directories, which contain references to other computer files, and possibly other directories. The term "folder" is often used to describe directories on computers.
 - term: dirty
```

Comment:
- term: directory
  description: >-
    Computer files are cataloged in directories, which contain references to other computer files, and possibly other directories. The term "folder" is often used to describe directories on computers.

Reply:
data/glossaries/external.yml